interview-questions.md

```diff
@@ -8,50 +8,80 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: A parameter is how we can make dynamic funcitons. We give the funciton value that can goes inside of parentheses in our arrow function set up. This value we can name as whatever we want but best pracitce is the type of data to be used. This is called the parameter. The argument is the actually data we want to pass in when running the code. This is usually found at the end of the funciton contained in the console.log line of code. 
 
-Researched answer:
+Researched answer: The variables that are used when a funcition is created are know as the parameters. The data or values that are used when the function is called are known as arguments.
+
```

Comment:
Don't forget that research is an opportunity to expand you knowledge on a topic. Don't be afraid to dig a little deeper.